am-lib/src/main/java/uk/gov/hmcts/reform/amlib/db/migration/V3__Create_default_roles_config_tables.sql

```diff
@@ -0,0 +1,47 @@
+CREATE TYPE "SecurityClassification" AS ENUM ('Public', 'Private', 'Restricted');
```

Comment:
Interesting approach with this enum as sql type. Here I've found an article that are evaluating this approach https://softwareengineering.stackexchange.com/a/305153